sdk/tables/aztables/README.md

```diff
@@ -0,0 +1,174 @@
+# Azure Tables client library for Go
+
+Azure Table storage is a service that stores large amounts of structured NoSQL data in the cloud, providing 
+a key/attribute store with a schema-less design. 
+
+Azure Cosmos DB provides a Table API for applications that are written for Azure Table storage that need premium capabilities like:
+
+- Turnkey global distribution.
+- Dedicated throughput worldwide.
+- Single-digit millisecond latencies at the 99th percentile.
+- Guaranteed high availability.
+- Automatic secondary indexing.
+
+The Azure Tables client library can seamlessly target either Azure Table storage or Azure Cosmos DB table service endpoints with no code changes.
+
+## Getting started
+
+### Install the package
+Install the Azure Tables client library for Go :
+
+### Prerequisites
+* An [Azure subscription][azure_sub].
+* An existing Azure storage account or Azure Cosmos DB database with Azure Table API specified.
+
+If you need to create either of these, you can use the [Azure CLI][azure_cli].
+
+#### Creating a storage account
+
+Create a storage account `mystorageaccount` in resource group `MyResourceGroup` 
+in the subscription `MySubscription` in the West US region.
+
+
+#### Creating a Cosmos DB
+
+Create a Cosmos DB account `MyCosmosDBDatabaseAccount` in resource group `MyResourceGroup` 
+in the subscription `MySubscription` and a table named `MyTableName` in the account.
+
+
+### Authenticate the Client
+
+Learn more about options for authentication _(including Connection Strings, Shared Key, and Shared Key Signatures)_ [in our samples.]()
+
+## Key concepts
+
+- `TableServiceClient` - Client that provides methods to interact at the Table Service level such as creating, listing, and deleting tables
+- `TableClient` - Client that provides methods to interact at an table entity level such as creating, querying, and deleting entities within a table.
+- `Table` - Tables store data as collections of entities.
+- `Entity` - Entities are similar to rows. An entity has a primary key and a set of properties. A property is a name value pair, similar to a column.
+
+Common uses of the Table service include:
+
+- Storing TBs of structured data capable of serving web scale applications
+- Storing datasets that don't require complex joins, foreign keys, or stored procedures and can be de-normalized for fast access
+- Quickly querying data using a clustered index
+- Accessing data using the OData protocol and LINQ filter expressions
+
+### Thread safety
+We guarantee that all client instance methods are thread-safe and independent of each other ([guideline](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-service-methods-thread-safety)). This ensures that the recommendation of reusing client instances is always safe, even across threads.
```

Comment:
If we want to link to this, should it instead be a section we discuss and add to the Go guidelines? In order to consider and add Go-specific conventions. (NOTE: I havent clicked on the link itself yet)

Reply:
Thanks - Yes, this was just an initial copy/paste from .NET. I'll remove for now unless we have a go equivalent statement